### ISSUE TITLE
[Hosted Che] Commands failing in NodeJS Angular workspace

### ISSUE THREAD

**Issue by Katka92** [eclipse-che/che#17724]
### Describe the bug
The command `yarn install` is failing. As a consequence, no other command runs successfully.

The workspace with **non-ephemeral** storage is failing on: `An unexpected error occurred: "EEXIST: file already exists, mkdir '/projects/angular-realworld-example-app/node_modules/make-dir'".` (see screenshot 1)

The workspace with **ephemeral** storage is failing on: 
`` which is caused by disk quota exceeded issue - I verified that by this after failed `yarn install`:
```
bash-4.4 /projects $ echo "test" > /project/foo
bash: /project/foo: No such file or directory
bash-4.4 /projects $ echo "test" > /projects/foo
bash: /projects/foo: Disk quota exceeded
bash-4.4 /projects $ 

```
(see screenshot 2)

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce

1. Go to `Get Started` page, `Get Started` tab
2. Check `Temporry Storage` that is set to proper flow that you want to test
3. Click `Nodejs Angular Web Application`
4. Run command `yarn install` 

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [x] Openshift (include output of `oc version`) OSD v4
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Screenshots
1:
![Screenshot from 2020-08-27 12-08-15](https://user-images.githubusercontent.com/29278800/91427521-55117a00-e85e-11ea-81ff-428eb69c8968.png)
2:
![Screenshot from 2020-08-27 12-16-56](https://user-images.githubusercontent.com/29278800/91430627-7cb51200-e85f-11ea-9eff-d24b03ac123d.png)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [x] I don't know - Hosted Che

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [x] other: please specify - Hosted Che

**Comment by ScrewTSW** [comment]
Still present in che 7.21.2
```
error Could not write file "/projects/angular-realworld-example-app/yarn-error.log": "errmap.get is not a function or its return value is not iterable"
error An unexpected error occurred: "Unknown system error -122: Unknown system error -122, copyfile '/home/user/.cache/yarn/v4/npm-http-proxy-1.17.0-7ad38494658f84605e2f6db4436df410f4e5be9a/node_modules/http-proxy/coverage/lcov-report/lib/http-proxy/index.html' -> '/projects/angular-realworld-example-app/node_modules/http-proxy/coverage/lcov-report/lib/http-proxy/index.html'".
```
![image](https://user-images.githubusercontent.com/16451875/99253049-55436500-2810-11eb-93c6-50cf51dd53d8.png)

**Comment by ScrewTSW** [comment]
Still present in che 7.22.0

**Comment by ScrewTSW** [comment]
Issue is resolved in 7.23.1, closing
